src/Build.UnitTests/BackEnd/TaskHostConfiguration_Tests.cs

```diff
@@ -361,13 +361,8 @@ public void TestTranslationWithValueTypesInDictionary()
                 1,
                 Directory.GetCurrentDirectory(),
                 null,
-#if FEATURE_THREAD_CULTURE
```

Comment:
Maybe it's not worth doing this until we get the green light for removing ns13? If we can't remove ns13, we'll have to put this back. Same goes for the other removed flags.